### PR TITLE
Adds & updates postman calls for new verification code issue responses

### PR DESCRIPTION
**Pull request by dennisgove**

### CODE REVIEW

File: contact-tracing.postman_collection.json [hunk 1]
```diff
@@ -784,6 +784,97 @@
...
+							"path": [
+								"notify",
+								"positive"
+							]
+						}
+					},
+					"response": []
+				},
+				{
+					"name": "Issue Verification Code",
```
Comment (jackmurdoch) [inline contact-tracing.postman_collection.json]: Looks like you've duplicated the `Notify positive` request with a different name - any reason we'd need both in the collection? Feel free to just rename the existing request to `Issue verification code`

Comment (dennisgove) [reply contact-tracing.postman_collection.json]: Actually I intended to update the endpoint to `/issue`. Looks like I forgot to export from postman after making that change.

File: contact-tracing.postman_collection.json [hunk 2]
```diff
@@ -992,7 +998,10 @@
 									"  pm.expect(tv4.validate(json, schema)).to.be.true",
 									"})",
 									"",
-									"pm.environment.set('code', json.code.toString())"
```
Comment (jackmurdoch) [inline contact-tracing.postman_collection.json]: The existing test script was important as it saves the values required for the APHL supporting endpoints - could you revert please?

Comment (dennisgove) [reply contact-tracing.postman_collection.json]: It looks like that check didn't exist in the 'Notify Positive' call. As that and this both return the exact same response I've added it in both.

Comment (jackmurdoch) [reply contact-tracing.postman_collection.json]: The `Notify positive` request (part of the old upload process) needs to generate the `uploadHash` to pass to `/exposures/verify`. However, the `Issue code` (part of the APHL supporting process) just needs to pass along the `code`  to `/verify`. The only changes that need to happen to the test scripts are the addition of `smsSent` to the response schema.

**Review by jackmurdoch — APPROVED** [review]

File: contact-tracing.postman_collection.json (hunk 2 above)
Comment (dennisgove) [reply contact-tracing.postman_collection.json]: I'm not sure I follow. They both execute the same code and return the same response. Shouldn't we be verifying all the expected values are returned?

**Pull request merged**